src/bridge/layouts/mod.rs

```diff
@@ -123,6 +132,36 @@ pub fn produce_neovim_keybinding_string(
     }
 }
 
+#[cfg(feature = "glfw")]
+pub fn produce_neovim_keybinding_string(
+    keycode: Option<Keycode>,
+    keytext: Option<String>,
+    modifiers: Option<Mod>,
+) -> Option<String> {
+    let mut shift = false;
+    let mut ctrl = false;
+    let mut alt = false;
+    let mut gui = false;
+    if let Some(modifiers) = modifiers {
+        shift = modifiers.contains(Mod::Shift);
```

Comment:
This is usually handled using bit shifts, so there isn't a need for an entire vector to handle the state. Is that how the modifier is implemented?

Reply:
Actually I did not completely get what you mean ? I just reference the code from both sdl2 and winit

Reply:
I double checked. This is fine. I thought this was self-implemented.